.github/workflows/release.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     - name: Sign the images with Cosign
       env:
         COSIGN_EXPERIMENTAL: 1
-      run: cosign sign $DOCKER_REGISTRY/${{ matrix.component }}:$TAG
+      run: cosign sign "$DOCKER_REGISTRY"/${{ matrix.component }}:"$TAG"
```

Comment:
we should probably quote the whole string -- if matrix.component has a space, this would break. Probably a few other instances of this.

Reply:
Done! Added quotes for the whole string in most places where this could occur.